integration_tests/tests/match/match.cy.ts

```diff
@@ -207,4 +209,46 @@ context('Placement Requests', () => {
     // Then I should see a success message on the list page
     Page.verifyOnPage(ListPage)
   })
+
+  it('allows me to book a space', () => {
+    // Given I am signed in as a cru_member
+    signIn(['cru_member'])
+
+    const premisesName = 'Hope House'
+    const premisesId = 'abc123'
+    const apType = 'normal'
+    const duration = 15
+    const startDate = '2024-07-23'
+
+    // And there is a placement request waiting for me to match
+    const person = personFactory.build()
+    const essentialCriteria = ['isPIPE', 'acceptsHateCrimeOffenders'] as Array<PlacementCriteria>
```

Comment:
We could type this instead of casting it, same with the desirable criteria below